CONTRIBUTING.md

```diff
@@ -17,21 +17,60 @@ If you are here to suggest a feature, first create an issue if it does not alrea
 If you have been assigned to fix an issue or develop a new feature, please follow these steps to get started:
 
 - Fork this repository.
-- Install dependencies by running `$ pnpm install`.
-  - We use [pnpm](https://pnpm.io/) v8 for package management (run `corepack enable && corepack prepare` in case of pnpm-related issues).
-  - We use [nvm](https://github.com/nvm-sh/nvm) to manage node versions - please make sure to use the version mentioned in `.nvmrc` (run `nvm use`).
-- Build all packages using `pnpm build:all`.
-- Run development server using `pnpm run watch`.
+- Install dependencies
+
+  ```bash
+  pnpm install
+  ```
```

Comment:
When using bash, I thought it would be good to make it easy to copy bash commands.